governator-test-junit/src/main/java/com/netflix/governator/guice/test/ModulesForTesting.java

```diff
@@ -43,5 +43,7 @@
 public @interface ModulesForTesting {
```

Comment:
looking at the style of other JUnit annotations (@Test, @Before, @BeforeClass, @After, @ReplaceWithMock, etc), it's conspicuous to me that only @Test uses the 'test' word at all.  To cleanly extend their style with new annotations, I'd go with maybe @InjectorBuilder(modules=""), and @PropertyOverride rather than @TestPropertyOverride